locale/en/grid.po

```diff
@@ -698,6 +698,12 @@ msgstr "{$userGroupName} role assigned to {$stageName} stage."
 msgid "grid.userGroup.unassignedStage"
 msgstr "{$userGroupName} role unassigned from {$stageName} stage."
 
+msgid "grid.publication.itemVoR"
+msgstr "VoR"
```

Comment:
Would abbreviating this present any issues for translations and/or users understanding what it means? I wonder if it should be written out, even though it's longer to display in the grid.